gcsweb/build/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
why?  does sh not exist?  I would only say "bash" if we need bash-isms.

Reply:
The golang image links /bin/sh to dash, which doesn't support things like `set -o pipefail` (apparently).